### PR TITLE
Update UR to include latest fixes

### PR DESCRIPTION
**Pull request by smaslov-intel**

### CODE REVIEW

**Comment by smaslov-intel** [comment]
@intel/dpcpp-l0-pi-reviewers : please review
@intel/llvm-gatekeepers : please merge

File: sycl/plugins/unified_runtime/CMakeLists.txt [hunk 1]
```diff
@@ -4,7 +4,7 @@ if (NOT DEFINED UNIFIED_RUNTIME_LIBRARY OR NOT DEFINED UNIFIED_RUNTIME_INCLUDE_D
   include(FetchContent)
 
   set(UNIFIED_RUNTIME_REPO "https://github.com/oneapi-src/unified-runtime.git")
-  set(UNIFIED_RUNTIME_TAG v0.7)
+  set(UNIFIED_RUNTIME_TAG 47b63869defab0686c58197271050d77ecf3287a)
```
Comment (kbenzie) [inline sycl/plugins/unified_runtime/CMakeLists.txt:7]: I've created [Prerelease v0.7.1](https://github.com/oneapi-src/unified-runtime/releases/tag/v0.7.1) pointing to this commit:

```suggestion
  set(UNIFIED_RUNTIME_TAG v0.7.1)
```

**Review by againull — APPROVED** [review]

**Comment by smaslov-intel** [comment]
> lit.py: /__w/llvm/llvm/llvm/sycl/test-e2e/lit.cfg.py:423: error: Cannot detect device aspect for ext_oneapi_hip:gpu
> lit.py: /__w/llvm/llvm/llvm/sycl/test-e2e/lit.cfg.py:423: error: Cannot detect device aspect for ext_oneapi_level_zero:gpu
> stdout:
>
> Platforms: 0
default_selector()      : No device of requested type available. -1 (PI_ERRO...
accelerator_selector()  : No device of requested type available. -1 (PI_ERRO...
cpu_selector()          : No device of requested type available. -1 (PI_ERRO...
gpu_selector()          : No device of requested type available. -1 (PI_ERRO...
custom_selector(gpu)    : No device of requested type available. -1 (PI_ERRO...
custom_selector(cpu)    : No device of requested type available. -1 (PI_ERRO...
custom_selector(acc)    : No device of requested type available. -1 (PI_ERRO...

@intel/llvm-gatekeepers : is this a known issue?

**Comment by aelovikov-intel** [comment]
> @intel/llvm-gatekeepers : is this a known issue?

Known but not understood. The system has this state:

> $ rocminfo
ROCk module is loaded
hsa api call failure at: /long_pathname_so_that_rpms_can_package_the_debug_info/src/rocminfo/rocminfo.cc:1166
Call returned HSA_STATUS_ERROR_OUT_OF_RESOURCES: The runtime failed to allocate the necessary resources. This error may also occur when the core runtime library needs to spawn threads or create internal OS-specific events.

and only reboot helps (which I've just performed).

**Comment by aelovikov-intel** [comment]
Failure on Intel GPU isn't related to this PR and earlier revision had successful results on this target. Also this fix is urgent and risks/benefits tell me to merge it.

**Pull request merged**